### DISCUSSION TITLE
Permanent-Link zum neuesten Snapshot

### DISCUSSION THREAD

**Discussion by jp112sdl** [OpenCCU/OpenCCU#1557] [🇩🇪 Generelle Diskussionen]
Im Rahmen der CI meines JP-HB-Devices-Addon würde ich gern einen Workflow erstellen, um automatisiert zu testen, ob sich noch alle Patche auf dem finalen Auslieferungs-Image anwenden lassen.

Ich hänge momentan an der Stelle, wie ich über eine fixe URL den jeweils neuesten/letzten Build bekomme.
_Im Idealfall die `.ccu3.tgz`, da sich das darin enthaltene ext4-Image direkt mounten lässt._

Ließe sich hier evtl. ein Link á la
`https://github.com/jens-maus/RaspberryMatic/releases/download/snapshots/latest-ccu3.tgz`
erstellen?

**Comment by jens-maus** [comment]
Nun, solche cross-repository workflows sind immer so ne Sache bei GitHub und nicht so einfach umzusetzen. Einen solchen `latest-ccu3.tgz` Link kann ich leider nicht einfach so im GitHub anlegen lassen. Da müsstest du schon auf deiner Seite (in deinem Repo) dir irgendeine Logik ausdenken die z.B. die releases Liste des RaspberryMatic GitHub Projektes irgendwie durchgeht um dann das neueste Snapshot-Image zu finden. Es gibt da ja public GitHub actions die es einem erlauben die release liste eines Projektes aufzulisten. so etwas müsstest du als Ansatz verwenden.

**Reply by jp112sdl** [reply]
>  Da müsstest du schon auf deiner Seite (in deinem Repo) dir irgendeine Logik ausdenken die z.B. die releases Liste des RaspberryMatic GitHub Projektes irgendwie durchgeht um dann das neueste Snapshot-Image zu finden

Großes Dankeschön an @psi-4ward für die Umsetzung mittels 
```BASH
RM_NIGHTLY_URL=$(curl -sSL -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/jens-maus/RaspberryMatic/releases/22744592/assets | jq -r '.[] | select(.name | endswith("ccu3.tgz")) | .browser_download_url')
```
